src/analysis/processing/qgsalgorithmgrid.cpp

```diff
@@ -0,0 +1,527 @@
+/***************************************************************************
+                         qgsalgorithmgrid.cpp
+                         ---------------------
+    begin                : August 2019
+    copyright            : (C) 2019 by Clemens Raffler
+    email                : clemens dot raffler at gmail dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+//Disclaimer:This feature was developed by: Michael Minn, 2010
+
+#include "qgsalgorithmgrid.h"
+#include "qgswkbtypes.h"
+#include "qgsvectorlayer.h"
+
+///@cond PRIVATE
+
+QString QgsGridAlgorithm::name() const
+{
+  return QStringLiteral( "creategrid" );
+}
+
+QString QgsGridAlgorithm::displayName() const
+{
+  return QObject::tr( "Create Grid" );
```

Comment:
```suggestion
  return QObject::tr( "Create grid" );
```

(following algorithm title conventions)